solr/core/src/test/org/apache/solr/cloud/OverseerSolrResponseUnsafeSerializationTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.solr.cloud;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+public class OverseerSolrResponseUnsafeSerializationTest extends OverseerSolrResponseTest {
+  
+  @BeforeClass
+  public static void setUpClass() {
+    System.setProperty("solr.unsafeOverseerResponseSerilization", "true");
+    System.setProperty("solr.unsafeOverseerResponseDeserilization", "true");
```

Comment:
Is there a use-case for two separate properties or should it be folded into one?

Reply:
I originally had a single property, but then I was thinking of the rolling restart case. We have the Overseer writing and other nodes consuming this data. If it's a single property there is no way to make the Overseer start writing Javabin while the consumers can consume both. An alternative is to only have `solr.unsafeOverseerResponseDeserilaization`, the catch there is that the Overseer needs to be the last one to be upgraded, otherwise it'll start writing a format that others may not be able to read. Other idea?

Reply:
You could have a single property with a string value. -Dsolr.unsafeOverseerResponse=[serialization,deserialization,all]

Or even parse a comma-separated list of values; this would take all out of the list.

Reply:
I don't think "serialization" makes a ton of sense. I'll have two options "true" (which turns on serialization and deserialization) and "deserialization", which turns on deserialization only.